fabfile.py

```diff
@@ -6,7 +6,7 @@
 PRODUCTION_APP_INSTANCE = 'wagtailio-production'
 STAGING_APP_INSTANCE = 'wagtailio-staging'
 
-LOCAL_MEDIA_FOLDER = '/vagrant/media'
+LOCAL_MEDIA_FOLDER = './media'
```

Comment:
this change will break the vagrant setup. will there be a follow up PR that removes vagrant support all together?

Reply:
I thought this would be OK as vagrant runs in `/vagrant`?

It's probably worth deleting Vagrant anyway so we don't have to duplicate everything